USAGE.md

```diff
@@ -26,9 +26,15 @@ having to answer the same question can really clog up the pipes for people who a
 
 ## From source
 
-1. Clone this repo: `git clone https://github.com/jabbink/PokemonGoBot.git && cd PokemonGoBot` or download the zip
-2. Run from terminal/cmd: `gradlew build`
-3. Rename `./config.properties.template` to `./config.properties`
-4. Modify `config.properties` as you please
-5. To run the bot directly from console run `gradlew run`
-6. :exclamation: If you use JetBrains IntelliJ, install the Lombok plugin and enable Settings -> Compiler -> Annotation Processors -> Enable annotation processing :exclamation:
+1. Make sure you have Oracle Java JDK 1.8 installed (`java -version` in a command line)
```

Comment:
javac -version works as well I think. And using that command here will probably stop issues from people that only installed the jre